Functions_and_Scope_Lab.js

```diff
@@ -0,0 +1,370 @@
+// # Function Exercises
+//
+// ## Question 1
```

Comment:
Overall good demonstration of functions and scope. I see some minor errors here but I think you understand concepts of functions and how to use them. Think about some of feedback and make sure to reflect that changes in the future. Keep up the good work!